### ISSUE TITLE
'Your developer account needs to be updated. Please visit Apple Developer Registration.' when trying to create certificates with match 

### ISSUE THREAD

**Issue by jonathanfoster** [fastlane/fastlane#18037]
TL;DR: Sign the terms and conditions on both [App Store Connect](https://appstoreconnect.apple.com/) and [Apple Developer](https://developer.apple.com).

After creating a new App Store Connect user, I was unable to create certificates with match using the new account. I received the error message "Your developer account needs to be updated. Please visit Apple Developer Registration." This is the same issue as https://github.com/fastlane/fastlane/issues/17072.

Unfortunately, https://github.com/fastlane/fastlane/issues/17072 was closed and it didn't mention a fix so I decided to document the fix here in case anyone else runs into the same problem. The root cause of the problem is the Apple ID account used has not signed terms and conditions on both App Store Connect and Apple Developer.

When creating a new user in App Store Connect, you're prompted to sign a service agreement there, but you're not prompted to sign the Apple Developer service agreement until you actually go to https://developer.apple.com. Once you do, make sure the account has App Manager role and access to certificates, identifiers, and profiles, and you're new account will be good to go.

**Comment by espadon8004** [comment]
Hi, same issue here,

I also end up on #17072 but this was closed without mentioning any fix.

**Comment by max-ott** [comment]
For reference: I've seen this error randomly popup with `fastlane` / `spaceship`, when using an old Apple ID. When I tried to login manually to https://developer.apple.com/account/ I always to redirected to https://developer.apple.com/apple-id with a similar site (with some variations) (ref #17072):
<img width="824" alt="Screenshot 2021-02-26 at 19 25 24" src="https://user-images.githubusercontent.com/6481790/109360078-e225a200-7886-11eb-8295-621afcc509df.png">
